### PR TITLE
add groupByExecutor main method doc

### PR DESCRIPTION
**Pull request by liutaohua**

### CODE REVIEW

File: docs/Documentation-CHN/SystemDesign/5-DataQuery/5-GroupByQuery.md [hunk 1]
```diff
@@ -111,17 +115,83 @@ private void calcFromBatch(BatchData batchData) throws IOException;
...
 //遍历并计算 page 中的数据
 private boolean readAndCalcFromPage() throws IOException, QueryProcessException;
 
 ```
-`GroupByExecutor` 中因为相同 `path` 的不同聚合函数使用的数据是相同的，所以 `calcFromBatch` 方法就是完成遍历所有聚合函数对 `BatchData` 的计算:
+`GroupByExecutor` 中因为相同 `path` 的不同聚合函数使用的数据是相同的，所以在入口方法 `calcResult` 中负责读取该 `Path` 的所有数据,
+取出来的数据再调用 `calcFromBatch` 方法完成遍历所有聚合函数对 `BatchData` 的计算。
+
+`calcResult` 方法的主要逻辑：
+```
```
Comment (qiaojialin) [inline docs/Documentation-CHN/SystemDesign/5-DataQuery/5-GroupByQuery.md]: add a blank line before ```

Comment (liutaohua) [reply docs/Documentation-CHN/SystemDesign/5-DataQuery/5-GroupByQuery.md]: fixed

File: docs/Documentation-CHN/SystemDesign/5-DataQuery/5-GroupByQuery.md [hunk 2]
```diff
@@ -111,17 +114,87 @@ private void calcFromBatch(BatchData batchData) throws IOException;
...
+      continue;
+    }
+    //如果不能使用 chunkStatistics 就需要使用page数据计算
+    if (readAndCalcFromPage()) {
+      return results;
+    }
+}
+```
+
+`readAndCalcFromPage` 方法是从当前打开的chunk中获取page的数据，并计算聚合结果，主要逻辑：
```
Comment (qiaojialin) [inline docs/Documentation-CHN/SystemDesign/5-DataQuery/5-GroupByQuery.md]: 加一个返回值的介绍

Comment (liutaohua) [reply docs/Documentation-CHN/SystemDesign/5-DataQuery/5-GroupByQuery.md]: yes, thanks for you advice

**Pull request merged**